changelogs/fragments/5706-add-builds-forks-container-registry.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - gitlab_project - add ``builds`` option (https://github.com/ansible-collections/community.general/pull/5706).
+  - gitlab_project - add ``forks`` option (https://github.com/ansible-collections/community.general/pull/5706).
+  - gitlab_project - add ``container_registry`` option (https://github.com/ansible-collections/community.general/pull/5706).
```

Comment:
Why not have just one entry which mentions all three options?

Reply:
Ok